edx_proctoring/static/proctoring/js/views/proctored_exam_info.js

```diff
@@ -112,16 +116,21 @@
 
         render: function() {
             var statusText = {};
+            var releaseDate;
+            var now = new Date();
             var data = this.model.toJSON();
             if (this.template) {
                 this.status = data.onboarding_status;
                 statusText = this.getExamAttemptText(data.onboarding_status);
+                releaseDate = new Date(data.onboarding_release_date);
                 data = {
                     onboardingStatus: statusText.status,
                     onboardingMessage: statusText.message,
+                    onboardingClosed: releaseDate > now,
```

Comment:
I feel like onboardingClosed could have two meanings - the onboarding is in the future or the onboarding window has already passed. Do we want to differentiate between these two?

Reply:
Yeah this could probably use a better variable name